core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -426,7 +425,14 @@ private class InProcessServerStream implements ServerStream {
 
       InProcessServerStream(MethodDescriptor<?, ?> method, Metadata headers) {
         statsTraceCtx = StatsTraceContext.newServerContext(
-            serverStreamTracerFactories, method.getFullMethodName(), headers);
+          serverStreamTracerFactories, method.getFullMethodName(), headers);
```

Comment:
Looks like this had proper indentation previously, and this changed it to be too little indentation.